.travis.yml

```diff
@@ -5,5 +5,10 @@ go:
 go_import_path: github.com/aws/amazon-ecs-agent
 
 install: make get-deps
+before_script:
+  # All the .go files in the agent, excluding vendor/ and testutils/ directories, and all *_test.go and *_mocks.go files
+  - GO_FILES=$(find ./agent -iname '*.go' -type f | grep -v /vendor/ | grep -v /testutils/ | grep -v _test\.go$ | grep -v _mocks\.go$)
```

Comment:
instead of `find`, it would be better to `go list ./agent/...`. Example: https://github.com/aws/amazon-ecs-agent/blob/73758e34dfc45d98ec6c3aaf623dd3bd8d6572b7/Makefile#L85

Reply:
"go list" lists the packages named by the import paths, but gocyclo needs local file/directory of go files. Looking through go list options to see if there's any I can use.

Reply:
yeah, that's not super optimal. I just assumed it'd be aware of packages.  I didn't want us to use `find` as the tool might not always be available in all environments. If it ran successfully in travis, it's probably fine for now.

Reply:
Bit messy but found a way to use "go list" instead of "find"